src/main/java/ch/njol/skript/expressions/ExprArgument.java

```diff
@@ -22,6 +22,7 @@
 import java.util.regex.MatchResult;
 
 import org.bukkit.event.Event;
+import org.bukkit.event.command.UnknownCommandEvent;
```

Comment:
Static imports on some version of a class that doesn't exist may throw unknown class found exception